index.rst

```diff
@@ -1,51 +1,387 @@
+
+
+Note that if a convergence test is used, it should only be allowed to exit the loop after a minimum number of iterations have passed.
+It will depend on how the convergence metric is calculated and the choice of initial solution, but the first iteration can show a slight degradation of convergence.
+
+
+Examples with simulated images
+------------------------------
+
+To test the above algorithm, I first ran it on images simulated using `StarFast <http://dmtn-012.lsst.io/en/latest/>`_.
+As shown in :numref:`fig-sim_image` below, these images contain a moderately crowded field of stars (no galaxies) with Kolmogorov double-gaussian PSFs, realistic SEDs, photon shot noise, and no other effects other than DCR.
+In this example, the model is built using three frequency planes and eight input simulations of the field, with airmass ranging between 1.0 and 2.0 (*not* including the simulated observation shown in :numref:`fig-sim_image`).
+I use :eq:`eqn-basic_template` to build a DCR-matched template for the simulated observation (:numref:`fig-sim_template`), and subtract this template to make the difference image (:numref:`fig-sim_template_diff`).
+For comparison, in :numref:`fig-sim_image_diff` I subtract a second simulated image generated with the same field 10 degrees closer to zenith.
+
+.. figure:: /_static/simulations/simulated_108_image.png
+   :name: fig-sim_image
+
+   Simulated g-band image with airmass 1.3
+
+.. figure:: /_static/simulations/simulated_image_108_template.png
+   :name: fig-sim_template
+
+   The DCR-matched template for the simulated image in :numref:`fig-sim_image`.
+
+.. figure:: /_static/simulations/simulated_image_108_template_difference.png
+   :name: fig-sim_template_diff
+
+   Difference image of :numref:`fig-sim_image` - :numref:`fig-sim_template`.
+
+.. figure:: /_static/simulations/simulated_image_108_110_difference.png
+   :name: fig-sim_image_diff
+
+   Difference image of :numref:`fig-sim_image` with another simulation of the same field 10 degrees closer to zenith (airmass 1.22).
+
+Examples with DECam images
+--------------------------
+
+For a more rigorous test, I have also built DCR-matched templates for `DECam HiTS <https://arxiv.org/abs/1609.03567>`_ observations, which were calibrated and provided by Francisco Förster. 
+For these images I used the implementation outlined above using the simplified equation :eq:`eqn-iterative_sum`, despite the images having variable seeing.
+Because I have not yet implemented :eq:`eqn-psf_iterative_sum` using measured PSFs for each image, I have simply excluded observations with PSF FWHWs greater than 4 pixels (2.5 - 4 pixel widths are common). 
+However, it should be noted that the images with PSFs at the larger end of that range are not as well matched by their templates, so restricting the input images going into the model to those with good seeing may not be sufficient in the future.
```

Comment:
Two things here.
1. I think you need to be clearer about what the differencing algorithm is.  Specifically this is *not* the stack algorithms, so describe what it *is*.
2. Those DECam images have cosmic rays in them.  It makes them look worse than they should.  I'm not sure what to do about that.  The stack pipelines will remove those, so you could try to use Meredith's stuff to reduce these images before diffing them.